### PR TITLE
Added main.py

### PR DESCRIPTION
**Pull request by joe1606**
Pipeline can be run though a single file now and without potential circular importing

### CODE REVIEW

**Comment by joe1606** [comment]
closes #66

File: pipeline/main.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    try:
+        extracted_data = extract_process()
+    except Exception as e:
+        logging.error(f'Error during extraction: {e}')
+
+    try:
+        transformed_data = transform_process(extracted_data)
+    except Exception as e:
+        logging.error(f'Error during transform: {e}')
```
Comment (laUmar123) [inline pipeline/main.py:25]: For this I'd say add a return statement if there is failure, for example:
`try:
        extracted_data = extract_process()
  except Exception as e:
        logging.error(f'Error during extraction: {e}')
        return
`
That way it doesn't continue when it's not necessary

**Review by laUmar123 — CHANGES_REQUESTED** [review]
Decent work again, please consider the changes I've mentioend

**Review by laUmar123 — DISMISSED** [review]
Please remove the unnecessary returns

**Review by laUmar123 — APPROVED** [review]
Happy with this now

**Pull request merged**